### PR TITLE
add signUp

### PR DESCRIPTION
**Pull request by manema**
- Added Sign up page.
- Added Sign up css page. 
- Added a text constraint on constraints file.
Trello card: https://trello.com/c/BbjU1Gr5/2-5-as-a-user-i-should-be-able-to-sign-up-so-my-account-is-created-in-target-mvd

### CODE REVIEW

File: src/components/user/SignUpForm.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import React from 'react';
```
Comment (fedeagripa) [inline src/components/user/SignUpForm.js]: Remember to remove commented code

File: src/components/user/SignUpForm.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+/*
+const message =  defineMessages ({
+    email: { id: 'login.form.email'},
+    password: { id: 'login.form.password'},
+    passwordConfirmation: { id: 'login.form.passwordConfirmation'}
+});
+*/
+
+const sexs = [{ sex: 'Male', value: 'male' },
```
Comment (fedeagripa) [inline src/components/user/SignUpForm.js]: Indentation:
```
const sexs = [{ sex: 'Male', value: 'male' },
              { sex: 'Female', value: 'female' },
              { sex: 'Other', value: 'other' }]
```

File: src/components/user/SignUpForm.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+              component={DropdownList}
+              data={sexs}
+              valueField="value"
+              textField="sex"
+              defaultValue='male'
+            />
+          </div>
+          <div>
+            <br />
+            <center><button type="submit" id="signupbtn">Sign up</button></center>
```
Comment (fedeagripa) [inline src/components/user/SignUpForm.js]: be consistent with camel or snake case

File: src/locales/en.js [hunk 4]
```diff
@@ -14,12 +14,15 @@ export default {
...
 
   // errors
   'email.presence': 'You must enter an email to continue',
   'email.invalid': 'You must enter a valid email',
   'password.presence': 'You must enter a password to continue',
   'passwordConfirmation.presence': 'You must enter a password confirmation to continue',
   'passwordConfirmation.equality': 'Your password confirmation must be equal to the password',
+  'name.presence': 'You must enter a name to continue',
+  'name.valid': 'Your name should contain only lowercase letters and uppercase letters',
+  'sex.presence': 'Your must select your sex to continue',
```
Comment (fedeagripa) [inline src/locales/en.js]: As dropdown have default value, does it make sense to have this?

File: src/locales/es.js [hunk 5]
```diff
@@ -19,12 +19,15 @@ export default {
...
   // errors
   'email.presence': 'Debe ingresar un email para continuar',
   'email.invalid': 'Debe ingresar un email válido',
   'password.presence': 'Debe ingresar una contraseña para continuar',
   'passwordConfirmation.presence': 'Debe confirmar su contraseña para continuar',
-  'passwordConfirmation.equality': 'Su confirmación de contraseña debe ser igual a la contraseña'
+  'passwordConfirmation.equality': 'Su confirmación de contraseña debe ser igual a la contraseña',
+  'name.presence': 'Debe ingresar un nombre para cotinuar',
+  'name.valid': 'Tu nombre deberia contener solo letras minusculas y mayusculas',
+  'sex.presence': 'Debe seleccionar su sexo para continuar',
```
Comment (fedeagripa) [inline src/locales/es.js]: Same here

File: src/styles/loginPage.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
+@import 'settings';
+
+#home-menu-open {
```
Comment (fedeagripa) [inline src/styles/loginPage.css]: are you using this?

File: src/styles/signupPage.css [hunk 7]
```diff
@@ -0,0 +1,76 @@
+@import 'settings';
+
+#signup_menu {
+  display: flex;
+  /* align-items: stretch; */
```
Comment (fedeagripa) [inline src/styles/signupPage.css]: please remove comment

File: src/styles/signupPage.css [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+#signup_title {
+  font-family: "Open Sans";
+  font-size: 27px;
+  font-weight: bold;
+}
+
+#signupbtn {
+  background-color: #000000;
```
Comment (fedeagripa) [inline src/styles/signupPage.css]: for simple colors please use it with words its more clear that way (black)

File: src/styles/signupPage.css [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  height: 768px;
+}
+
+#signup_title {
+  font-family: "Open Sans";
+  font-size: 27px;
+  font-weight: bold;
+}
+
+#signupbtn {
```
Comment (fedeagripa) [inline src/styles/signupPage.css]: be consistente with sneak case

File: src/styles/signupPage.css [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+  font-size: xx-small;
+}
+
+.dropbtn {
+  padding: 16px;
+  font-size: 16px;
+  border: 2px solid #cacaca;
+  background-color: #f9f9f9;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
```
Comment (fedeagripa) [inline src/styles/signupPage.css]: no need of px for 0

File: src/styles/loginPage.css (hunk 6 above)
Comment (manema) [reply src/styles/loginPage.css]: I will use it in the next sprint

**Pull request merged**